images/syncer/Dockerfile

```diff
@@ -48,6 +48,9 @@ RUN go build -a -ldflags="-w -s -extldflags=static -X sigs.k8s.io/vsphere-csi-dr
 ################################################################################
 FROM ${BASE_IMAGE}
 
+RUN tdnf -y remove toybox
```

Comment:
Why are you removing it? Wouldn't it give us some default utils in the container image that may be necessary?

Reply:
we have removed the toybox package from the base driver image as well - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/images/driver-base/Dockerfile#L18

as per the description of toybox package. We do not need this utility in our driver and syncer image so we are removing it.

```
Description   : Toybox combines common Linux command line utilities together into a single
BSD-licensed executable that's simple, small, fast, reasonably
standards-compliant, and powerful enough to turn Android into a development
environment.
```